src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -1,19 +1,23 @@
 package bio.terra.service.dataset;
 
-import bio.terra.service.iam.AuthenticatedUserRequest;
-import bio.terra.service.dataset.flight.create.DatasetCreateFlight;
-import bio.terra.service.dataset.flight.delete.DatasetDeleteFlight;
-import bio.terra.service.dataset.flight.ingest.DatasetIngestFlight;
 import bio.terra.common.MetadataEnumeration;
+import bio.terra.model.AssetModel;
 import bio.terra.model.DatasetModel;
 import bio.terra.model.DatasetRequestModel;
 import bio.terra.model.DatasetSummaryModel;
 import bio.terra.model.DeleteResponseModel;
 import bio.terra.model.EnumerateDatasetModel;
 import bio.terra.model.IngestRequestModel;
+import bio.terra.service.dataset.flight.create.AddAssetSpecFlight;
```

Comment:
Is an extra level of package needed? I kinda like opening the flight package and seeing everything...